File: TwoDeeSharp/Game.cs
using System.Html;

namespace TwoDeeSharp
{
    public class Game
    {
        public Game()
        {
            var canvasBgElement = (CanvasElement) Document.GetElementById("gameBG");
            var canvasSpritesElement = (CanvasElement) Document.GetElementById("gameSprites");
            var canvasFgElement = (CanvasElement) Document.GetElementById("gameFG");

            ScreenModel = new ScreenModel(canvasBgElement, canvasSpritesElement, canvasFgElement);

            Window.AddEventListener("resize", resizeCanvas, false);
            resizeCanvas();

            CurrentGameView = new GameView(ScreenModel, new GameModel(StaticData.FakeGameData[0]));
        }

        public ScreenModel ScreenModel { get; set; }
        public GameView CurrentGameView { get; set; }

        private void resizeCanvas()
        {
            var w = (uint) Window.InnerWidth;
            var h = (uint) Window.InnerHeight;
            ScreenModel.CanvasBgElement.Width = w;
            ScreenModel.CanvasBgElement.Height = h;
            ScreenModel.CanvasFgElement.Width = w;
            ScreenModel.CanvasFgElement.Height = h;
            ScreenModel.CanvasSpritesElement.Width = w;
            ScreenModel.CanvasSpritesElement.Height = h;
        }
    }
}

File: game/game.js



function Game() {
    var canvasBgElement = document.getElementById('gameBG');
    var canvasSpritesElement = document.getElementById('gameSprites');
    var canvasFgElement = document.getElementById('gameFG');

    var screenModel=this.screenModel = new ScreenModel(canvasBgElement, canvasSpritesElement, canvasFgElement);


    window.addEventListener('resize',resizeCanvas, false);

    resizeCanvas();
    function resizeCanvas() {
        var w = window.innerWidth;
        var h = window.innerHeight;
        screenModel.canvasBgElement.width = w;
        screenModel.canvasBgElement.height = h;
        screenModel.canvasFgElement.width = w;
        screenModel.canvasFgElement.height = h;
        screenModel.canvasSpritesElement.width = w;
        screenModel.canvasSpritesElement.height = h;
    }

    this.currentGameView = new GameView(this.screenModel, new GameModel(window.FakeGameData[0]));


}


